Makefile

```diff
@@ -0,0 +1,4 @@
+.PHONY: build
+build:
```

Comment:
Let's change this to something more useful/descriptive at some later stage, not important for now though